docx/oxml/text/paragraph.py

```diff
@@ -17,11 +17,28 @@ class CT_P(BaseOxmlElement):
     pPr = ZeroOrOne('w:pPr', successors=('w:bookmarkEnd',))
     r = ZeroOrMore('w:r', successors=('w:bookmarkEnd',))
     bookmarkEnd = ZeroOrMore('w:bookmarkEnd')
+    hyperlink = ZeroOrMore('w:hyperlink')
 
     def _insert_pPr(self, pPr):
         self.insert(0, pPr)
         return pPr
 
+    def add_hyperlink(self, reference, text):
```

Comment:
Another small suggestion, I think the API would be better and it would increate the readability if the `text` would be the first parameter and the `reference` second. Check the tests and let me know what you think

Reply:
Ok, `text` can be the first parameter.